src/clarity-angular/dropdown/_dropdown.clarity.scss

```diff
@@ -84,6 +84,18 @@ $dropdown-white: clr-getColor(lightest);
             visibility: hidden;
             z-index: map-get($clr-layers, dropdown-menu);
 
+            .dropdown-menu {
```

Comment:
oh sorry, I think misunderstood the purpose of it. I guess this is a temporary hack that overrides the style that should take effects in 0.9.0. If then, we could make use of `& {` instead of using the class name again?